### PR TITLE
LoadCape: Example for toggling each load

### PR DESCRIPTION
**Pull request by lorforlinux**
This python example allows the user to toggle each load one by one.

### CODE REVIEW

**Comment by lorforlinux** [comment]
@MarkAYoder @jadonk please review my code :)

**Comment by MarkAYoder** [comment]
@lorforlinux I don't have` /sys/class/leds/load-sink` on my bones.  What is it?

**Comment by lorforlinux** [comment]
@MarkAYoder you need to update the uEnv.txt to install Load Cape overlay (BBORG_LOAD-00A2.dtbo) during boot.

File: Capes/loadCape/toggleEach.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```
Comment (jadonk) [inline Capes/loadCape/toggleEach.py:1]: If overlays are required to be loaded or hardware is to be installed, please add comments to that effect.

**Review by jadonk — CHANGES_REQUESTED** [review]
Please add some more comments in the file.

File: Capes/loadCape/toggleEach.py (hunk 1 above)
Comment (jadonk) [reply Capes/loadCape/toggleEach.py:1]: Oh, and send the pull request to v2020.08, not v2020.01!

**Comment by lorforlinux** [comment]
I am closing this PR will create a new one ;)

**Pull request closed without merging**